Gemfile

```diff
@@ -46,8 +46,11 @@ gem 'rspec-rails', group: [:development, :test]
 gem 'capybara', group: [:development, :test]
 gem 'launchy', group: [:development, :test]
 gem 'factory_girl', group: [:development, :test]
+gem 'faker', group: [:development, :test]
+gem 'factory_girl_rails', group: [:development, :test]
 gem 'valid_attribute', group: [:development, :test]
 gem 'shoulda-matchers', group: [:development, :test], require: false
 gem 'devise'
 gem 'foundation-rails'
 gem 'coveralls', group: [:test], require: false
+gem 'pry-rails', group: [:development, :test]
```

Comment:
since your Gemfile is getting pretty hefty, you can clean them up instead of having bracketed environments on each line.

```ruby

group :development, :test do
  gem 'pry-rails'
  . 
  .
  .
end
```